README.rst

```diff
@@ -36,6 +36,12 @@ In order to authorize against Avaya OneCloud™️ CPaaS  services you'll have t
     
 .. code-block:: bash
 
+You can also change the base API url in /constants.go. The default is set to "https://api.zang.io".
+
+The base API URL for US(new) and EU deployments are:
+US: https://api-us.cpaas.avayacloud.com
+EU: https://api-eu.cpaas.avayacloud.com
```

Comment:
Why there is no '/v2' here ? but present in https://github.com/zang-cloud/zang-dotnet/pull/13/files

Reply:
Just make sure that '/v2' added internally by SDK. If yes. this is fine